integration-tests/testsetups/ocr.go

```diff
@@ -88,10 +91,25 @@ type OCRSoakTest struct {
 	reorgHappened              bool                  // flag to indicate if a reorg happened during the test
 	gasSpikeSimulationHappened bool                  // flag to indicate if a gas spike simulation happened during the test
 	gasLimitSimulationHappened bool                  // flag to indicate if a gas limit simulation happened during the test
+	chaosList                  []*k8schaos.Chaos     // list of chaos simulations to run during the test
+}
+
+type OCRSoakTestOption = func(c *OCRSoakTest)
+
+func WithChaos(chaosList []*k8schaos.Chaos) OCRSoakTestOption {
+	return func(c *OCRSoakTest) {
+		c.chaosList = chaosList
+	}
+}
+
+func WithNamespace(ns string) OCRSoakTestOption {
+	return func(c *OCRSoakTest) {
+		c.namespace = ns
+	}
 }
 
 // NewOCRSoakTest creates a new OCR soak test to setup and run
-func NewOCRSoakTest(t *testing.T, config *tc.TestConfig, forwarderFlow bool) (*OCRSoakTest, error) {
+func NewOCRSoakTest(t *testing.T, config *tc.TestConfig, forwarderFlow bool, opts ...OCRSoakTestOption) (*OCRSoakTest, error) {
```

Comment:
small extra: what about making `forwarderFlow` also an opt? would be easier to understand without checking this method's signature to check what that bool is about.

Reply:
@Tofel yes, this makes sense. Let me do it!

Reply:
@Tofel fixed in [Make forwarderFlow an opt](https://github.com/smartcontractkit/chainlink/pull/13624/commits/77c9fd7cebcb64c7e470acf14f0be65aaf0f7475)